perl/INSTALL.md

```diff
@@ -250,7 +250,8 @@ Database configurations are stored in YAML files in the `config` directory. By d
 
```

Comment:
I found a strikethrough at line 202 because of two ~s

hg19 (~/bystro/config/hg19.yml) and hg38 (~/bystro/config/hg38.yml),

Reply:
hm, I don't see this, could you show me?
<img width="2044" alt="Screenshot 2024-10-07 at 4 35 28 PM" src="https://github.com/user-attachments/assets/3173435e-e431-44dd-bcde-744292d683f0">